config/routes.rb

```diff
@@ -1,90 +1,77 @@
 Rails.application.routes.draw do
 
+  root "user_authentication#recommendations"
 
+  get ":user_id/recommendations" => "user_authentication#recommendations", as: :recommendations
 
   # Routes for the Rating resource:
 
-  root "user_authentication#feed"
-
   # CREATE
-  post("/insert_rating", { :controller => "ratings", :action => "create" })
+  post "/insert_rating" => "ratings#create", as: :new_rating
           
-  # READ
-  get("/ratings", { :controller => "ratings", :action => "index" })
-  
-  get("/ratings/:path_id", { :controller => "ratings", :action => "show" })
-  
-  # UPDATE
-  
-  post("/modify_rating/:path_id", { :controller => "ratings", :action => "update" })
-  
-  # DELETE
-  get("/delete_rating/:path_id", { :controller => "ratings", :action => "destroy" })
-
   #------------------------------
 
   # Routes for the Restaurant resource:
 
   # CREATE
-  post("/insert_restaurant", { :controller => "restaurants", :action => "create" })
+  post "/insert_restaurant" => "restaurants#create", as: :new_restaurant
           
   # READ
-  get("/restaurants", { :controller => "restaurants", :action => "index" })
+  get "/restaurants" => "restaurants#index", as: :restaurants
   
-  get("/restaurants/:path_id", { :controller => "restaurants", :action => "show" })
+  get "/restaurants/:id" => "restaurants#show", as: :restaurant
   
   # UPDATE
   
-  post("/modify_restaurant/:path_id", { :controller => "restaurants", :action => "update" })
+  post "/modify_restaurant/:id" => "restaurants#update", as: :update_restaurant
   
   # DELETE
-  get("/delete_restaurant/:path_id", { :controller => "restaurants", :action => "destroy" })
+  get "/delete_restaurant/:id" => "restaurants#destroy", as: :delete_restaurant
 
   #------------------------------
 
   # Routes for the Burrito resource:
 
   # CREATE
-  post("/insert_burrito", { :controller => "burritos", :action => "create" })
+  post "/insert_burrito" => "burritos#create", as: :new_burrito
           
   # READ
-  get("/burritos", { :controller => "burritos", :action => "index" })
   
-  get("/burritos/:path_id", { :controller => "burritos", :action => "show" })
+  get "/burritos/:id" => "burritos#show", as: :burrito
```

Comment:
nice refactor 👍🏽 

We can do even better if all the burrito routes are named RESTfully and conventionally by using [the `resources` method](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default).